cl/users/models.py

```diff
@@ -284,6 +280,11 @@ class BackoffEvent(AbstractDateTimeModel):
     next_retry_date = models.DateTimeField(
         help_text="The next retry datetime for exponential backoff events.",
     )
+    event_sub_type = models.SmallIntegerField(
+        help_text="The SES bounce subtype that triggered the object.",
+        choices=EMAIL_NOTIFICATIONS.TYPES,
+        default=EMAIL_NOTIFICATIONS.UNDETERMINED,
```

Comment:
Why do we assign a default value?  Is there a possibility we wouldn't receive an event_subtype?

Reply:
this maybe nitpicky - but im bothered by sub_type ... I think it should be subtype one word.  

I grepped the code and see it used both ways but @mlissner im sure you are going to agree. right.

Reply:
Either way seems ok to me, but I've been looking at it awhile.

Reply:
I changed the `event_sub_type` column for `notification_subtype` but only for the `BackoffEvent` model where this is a new column added in this PR, for `EmailFlag` didn't change it as we talked about as is an existing column in production.